### ISSUE TITLE
Instagram source connector throws error due to default clustering/partitioning by _airbyte_emitted_at

### ISSUE THREAD

**Issue by emilycogsdill** [airbytehq/airbyte#7937]
Airbyte version: 0.30.39-alpha
OS Version / Instance: Ubuntu VM / GCP n1-standard-2 (2 vCPUs, 7.5 GB memory)
Deployment: Docker
Source Connector and version: Instagram 0.1.9
Destination Connector and version: BigQuery 0.4.1
Severity: Medium
Step where error happened: dbt transformation (basic transformation, without customization)

I set up a fresh Instagram source connector today and the basic dbt transformation failed with the following error message: 
```
No matching signature for function TIMESTAMP_TRUNC for argument types: STRING, DATE_TIME_PART. Supported signatures: TIMESTAMP_TRUNC(TIMESTAMP, DATE_TIME_PART, [STRING]); TIMESTAMP_TRUNC(DATETIME, DATE_TIME_PART) at [5:16]
```

I did a little tinkering and figured out that this was being caused by the configuration generated in the dbt transformations, which attempts to cluster the database objects using `_airbyte_emitted_at`:

```
    cluster_by = "_airbyte_emitted_at",
    partition_by = {"field": "_airbyte_emitted_at", "data_type": "timestamp", "granularity": "day"},
```

Once I removed this from the code, the data sync succeeded. This leads me to believe that the default clustering and partitioning in the basic dbt transformation is generally problematic for Instagram. I haven't yet done further science to figure out what exactly to change to perform timestamp-based clustering, since it is not an immediate need of mine - but it seems the dbt transformations currently being generated by default are causing the errors I experienced.

**Comment by sherifnada** [comment]
@emilycogsdill this should be fixed on the latest version of bigquery, can you update your BQ destination connector and try again?

**Comment by emilycogsdill** [comment]
@sherifnada so sorry for the delay in getting to this, life got in the way! I can confirm that this is resolved when upgrading Airbyte to v0.32.0, which makes me suspect your hunch about the BQ destination connector was correct (although I neglected to test that specifically).

**Comment by sherifnada** [comment]
thanks for following up!
